.github/workflows/release-please.yml

```diff
@@ -11,15 +11,13 @@ jobs:
         id: release
         with:
           release-type: 'node'
-          target-branch: '${{ github.ref_name }}'
+          default-branch: '${{ github.ref_name }}'
```

Comment:
I didn't see an argument for `target-branch` in the config? https://github.com/google-github-actions/release-please-action/blob/main/action.yml

Reply:
The CLI renamed this param, but looks like you're right, the action uses `default-branch`.